wiki/Tournaments/GBC/GBC_2023_Autumn/en.md

```diff
@@ -0,0 +1,283 @@
+3. Both captains will alternate picking a beatmap from the mappool.
+4. Each captain must use `!roll` once in `#multiplayer`.
+   - The winner of the `!roll` could choose to pick first or ban first of the match. The other choose will be used by loser team. First pick team will also protect first, the others will ban first.
+5. After each pick, both captains **may** ban **one** of the opposing team members from playing the beatmap in a match, **except during tiebreakers**. A player cannot be banned during a match until they have played one beatmap.
+   - Any player can only be banned like this **once** per match.
+   - Only players who have already played in the match can be banned.
+   - The team that picks beatmaps first cannot ban players during the first 2 rounds.
+     - As a consequence of the earlier rules, no teams are allowed to ban players during the first round since no one has finished playing any beatmaps.
+   - Map picking and ban should be sent in the same message, if the ban is delayed, or the message cause a disagreement or misunderstanding, the referee has the right to ignore the ban.
+6. Teams will have 2 minutes to pick a beatmap and ban an opposing player, and 2 minutes to get ready. 
+   - A team can request a timeout when they are picking map. The timeout lasts 3 minutes and will be used after the map pick timer.
+   - No matter which team needs to timeout after pick map, it can only allowed with the consent of both captains.
+7. If a team takes more time than allowed, the procedures adopted will be as follows:
+   - Picking beatmaps and banning players should be sent **using a single message**. If a team continues to be slow at banning, referees have rights to **ignore the ban**.
+   - For the first occurrence:
+     - The team will receive a verbal warning from the referee.
+   - On subsequent occurrences:
+     - For bans of opposing players, the captain forfeits the ban.
+     - For map bans, the captain forfeits the ban.
+     - For map picks, a random map will be chosen from the mappool using `!roll X`, where X is the number of maps that were neither picked nor banned, excluding the tiebreaker.
+     - For a ready timer, the referee will issue the `!mp start 10` command, regardless of how many players from each team are present in the lobby, using `!mp kick` on any extra players for each team, starting from the top (i.e. the first valid player combination for each team will be forced to play the pick). The results for such games are to be taken as is.
+   - Repeat offenders may receive further sanctions from the host.
+8. Each team will receive **one** "tactical timeout" of **two minutes**, to be used as extra time to pick or ban a beatmap. *The tactical timeout is optional.*
+   - The tactical timeout may be called on a tiebreaker, provided that both teams' captains agree to it.
```

Comment:
this rule conflicts with rule 6. please remove one of them and adjust accordingly